images/openshift-kubernetes-nmstate-operator.yml

```diff
@@ -13,6 +13,7 @@ content:
 distgit:
   component: ose-kubernetes-nmstate-operator-container
   bundle_component: ose-kubernetes-nmstate-operator-bundle-container
+  branch: rhaos-{MAJOR}.{MINOR}-rhel-9
```

Comment:
But everything else here is still rhel8? :thinking:

Reply:
several were missing. no guarantee i got everything like this but at least for these three files the changes look complete.